modules/i2c/i2cs.h

```diff
@@ -0,0 +1,50 @@
+/** @defgroup i2c I2C
+ * @brief I2C module
+ */
+//@{
+/**
+ * @file
+ * @brief I2C definitions
+ */
+#ifndef I2CS_H__
+#define I2CS_H__
+
+#include <avr/io.h>
+#include <avarix/intlvl.h>
+#include <stdint.h>
+#include "i2c_config.h"
+
+/* Transaction status defines.*/
+#define I2CS_STATUS_READY                0
+#define I2CS_STATUS_BUSY                 1
+
+/* Transaction result enumeration */
+typedef enum {
+  I2CS_RESULT_UNKNOWN            = 0,
+  I2CS_RESULT_RECEIVED          ,
+  I2CS_RESULT_TRANSMIT          ,
+  I2CS_RESULT_OK                ,
+  I2CS_RESULT_BUFFER_OVERFLOW   ,
+  I2CS_RESULT_TRANSMIT_COLLISION,
+  I2CS_RESULT_BUS_ERROR         ,
+  I2CS_RESULT_FAIL              ,
+  I2CS_RESULT_ABORTED           
+} i2c_result_t;
+
+/*! \brief i2c slave driver struct.
```

Comment:
For consistency:
- use `/** @brief`
- Use I2C (not i2c) in comments (unless speaking of the module or a symbol name).

And as above, don't try align comments.